types/react-ga-tests.ts

```diff
@@ -0,0 +1,99 @@
+import * as ga from "react-ga";
```

Comment:
This makes sure the typescript bindings work, as the typescript compilation would fail if the typings were wrong.